components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1334,6 +1348,14 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
                   aer_asm_bnd_sw = 0
                end if
 
+               ! Check (and possibly clip) values before passing to RRTMGP driver
+               call handle_error(clip_values(cld_tau_gpt_sw,  0._r8, huge(cld_tau_gpt_sw), trim(subname) // ' cld_tau_gpt_sw', tolerance=1e-10_r8))
```

Comment:
Any evidence that cld_tau_gpt_sw and aer_tau_gpt_sw could get an infinite number? If certainly unlikely, the clipping calls for them can be safely removed.

Reply:
Not physically, this would be an indication that things went haywire somewhere. I wanted to keep these here for consistency, and to make sure we were handling the sanity checking ourselves before RRTMGP had to deal with it. This will catch cases where the tau values go to nan/inf before RRTMGP does.